File: lecture_001(Basic)/l_009_conversion.cpp
#include<iostream>
#include<string>
using namespace std;

int main() {
    //String to an Integer
    string str;
    getline(cin,str);  
    int i = stoi(str);
    cout<<i<<endl;

    //Integer to a string
    int m;
    cin>>m;
    string s = to_string(m);
    cout<<s<<endl;
}

File: lecture_001(Basic)/l_009_conversion.java
import java.util.Scanner;
public class l_009_conversion {
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args) {
    	//String to an Integer
        String str = scn.nextLine();
        int i = Integer.parseInt(str);
        System.out.println(i);
        
        //Integer to a String
        int m = scn.nextInt();
        String spd = Integer.toString(m);
        System.out.println(spd);
        
        //Integer to a String
        int p = scn.nextInt();
        String lol = String.valueOf(p);
        System.out.println(lol);
    }
}